test/integration/testStaticSite/families/greyjoy/members.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Grejoy Alive List</title>
+    <meta charset="utf-8"/>
+    <link rel="stylesheet" href="/dummy-base-url/assets/css/app.css">
+  </head>
+  <body>
+      <p><strong>Living Greyjoys</strong></p>
+    <ul>
+      <li>Theon Greyjoy</li>
+      <li>Euron Greyjoy</li>
+      <li>Yara Greyjoy - Unconfirmed</li>
```

Comment:
uhhhhh

Reply:
To be fair, at the end of the last season it wasn't clear what had happened to her.